src/intervals/trigonometric_functions.jl

```diff
@@ -4,9 +4,13 @@ half_pi(::Type{Float64})  = @floatinterval(pi/2)
 two_pi(::Type{Float64})   = @floatinterval(2pi)
 half_pi(::Type{BigFloat}) = @biginterval(pi/2)
 two_pi(::Type{BigFloat})  = @biginterval(2pi)
-
 half_pi{T<:AbstractFloat}(x::T) = half_pi(T)
 
+pi_interval{T<:Real}(::Type{T}) = get_pi(Float64)
```

Comment:
We should have either `get_pi` or `pi_interval`. This function should return the current value of `interval_parameters.pi`, and all of the other `pi` functions should be defined in terms of this.